service/raft/group0_state_machine.cc

```diff
@@ -266,15 +266,17 @@ future<> group0_state_machine::transfer_snapshot(raft::server_id from_id, raft::
     }
 
     std::optional<service::raft_topology_snapshot> topology_snp;
-    std::optional<service::raft_snapshot> auth_snp;
+    std::optional<service::raft_snapshot> raft_snp;
```

Comment:
nit: all snapshots are raft

Reply:
But service levels and the status are also included in this snapshot, so `raft_snp` fits better than `auth_snp`. And as I remember @piodul said other tables are going to be included there eventually.

Reply:
Yes - in fact, we are going to move the topology snapshot to the generic raft snapshot. See Gleb's `[PATCH] raft topology: drop RAFT_PULL_TOPOLOGY_SNAPSHOT RPC` on the scylladb-dev mailing list.